pytest_operator/plugin.py

```diff
@@ -403,13 +551,15 @@ async def run(self, *cmd, cwd=None, check=False, fail_msg=None):
 
         proc = await asyncio.create_subprocess_exec(
             *(str(c) for c in cmd),
-            cwd=str(cwd or "."),
+            stdin=asyncio.subprocess.PIPE if isinstance(stdin, bytes) else None,
```

Comment:
Should this fail if `not isinstance(stdin, bytes)` rather than silently work?  I think as is, this could make finding errors elsewhere in the code harder